lib/Doctrine/DBAL/Schema/MySqlSchemaManager.php

```diff
@@ -192,6 +192,14 @@ protected function _getPortableTableColumnDefinition($tableColumn)
 
         $column = new Column($tableColumn['field'], Type::getType($type), $options);
 
+        /**
+         * The key 'characterset' in $tableColumn is from MySql platform implementation
+         *
+         * @see MySqlPlatform::getListTableColumnsSQL()
+         */
```

Comment:
This doc block seems redundant. There are many more usages of `$tableColumn`'s keys in this method, and they all come from the same place.

If you still believe it needs better documentation, it should be documented at the `AbstractSchemaManager::_getPortableTableColumnDefinition()` or `MySqlSchemaManager::_getPortableTableColumnDefinition()`. Otherwise, please drop this.

Reply:
Dropped.